stories/carousel/carousel.stories.js

```diff
@@ -477,3 +476,100 @@ carouselNoNavigation.parameters = {
 `
     }
 };
+
+export const horizontalCarousel = () => `<div style="display: flex; flex-direction: column; align-items: center; background: #CCD1D1;">
+    <h4>Items in horizontal direction</h4>
+    <div
+        class="fd-carousel"
+        data-ride="carousel"
+        style="margin-bottom: 3rem; max-width: 60rem;">
+        <div class="fd-carousel__content fd-carousel__horizontal" style="text-align: center; padding: 1rem; min-height: 15.5rem;">
```

Comment:
It's wrong again

Reply:
have moved it inside __content. so the class name now is __content--horizontal.
have created class for --error-message, as it was getting broken after above change.